src/js/plugins/plugin.resume.js

```diff
@@ -44,10 +44,10 @@ BookReader.prototype.getResumeValue = function() {
  * Can be overriden for different implementation
  *
  * @param {Number} index leaf index
- * @param {String} cookieName
  */
-BookReader.prototype.updateResumeValue = function(index, cookieName='br-resume') {
+BookReader.prototype.updateResumeValue = function(index, cookieName) {
```

Comment:
Default parameter getting `null` value always which is setting into cookie as well. So we need to apply default with || operator to prevent the case when default parameter value coming `null`. @cdrini !

See reference: https://www.codereadability.com/javascript-default-parameters-with-or-operator/